src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+/**
+ * The class with deals with user interaction.
+ */
+public class Ui {
+    /**
+     * Returns a welcome message for when the user launch the application.
+     * @return A welcome message.
+     */
+    public String welcome() {
+        return "Hi, I'm Jamie.\nWhat do you want to do?"
+                + "\nFor dates and time, please enter in the format:"
+                + "\ndd/MM/yyyy HHmm eg. 29/10/2022 0000";
+    }
+
+    /**
+     * Returns the taskList in string form.
+     * @param tasks The taskList that is being printed.
+     * @return The tasks in the taskList in string form.
+     */
```

Comment:
I think there should be a line break after the description (line 18). The same goes for all the other JavaDoc comments.

Reply:
Thank you for letting me know about this.